packages/web-api/src/WebClient.spec.ts

```diff
@@ -69,7 +69,7 @@ describe('WebClient', () => {
 
     it('should succeed when constructing a class that extends WebClient', () => {
       assert.doesNotThrow(() => {
-        class X extends WebClient {}
+        class X extends WebClient { }
```

Comment:
Nit: Is this whitespace required or common convention (asking because I don't know)?
```suggestion
        class X extends WebClient {}
```